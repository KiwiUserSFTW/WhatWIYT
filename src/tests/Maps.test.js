import React from "react";
import renderer from "react-test-renderer";
import MapsPalete from "../components/Maps";
import TestProvider from './tests_tools/TestProvider'

test('Maps was correctly', () => {
    const tree = renderer
    .create(
    <TestProvider>
        <MapsPalete />
    </TestProvider>
    )
    .toJSON();

    expect(tree).toMatchSnapshot();
})