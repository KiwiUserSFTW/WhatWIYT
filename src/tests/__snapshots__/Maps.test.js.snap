// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Maps was correctly 1`] = `
<div
  className="MuiBox-root css-1so78ta"
>
  <div
    className="MuiGrid-root css-10dsau7-MuiGrid-root"
  >
     
  </div>
</div>
`;
